src/components/Nav/Nav.tsx

```diff
@@ -196,6 +194,14 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     ev.preventDefault();
     ev.stopPropagation();
   }
+
+  private _onNavLinkClicked(link: INavLink, ev: React.MouseEvent): void {
+    this._selectedKey = link.key;
+
+    if (link.onClick) {
+      link.onClick(ev);
```

Comment:
you man want to pass in the link as the 2nd param. The onClick definition could be:

onClick: (ev?: React.MouseEvent, link?: INavLink)